cmd/mount_all.go

```diff
@@ -68,37 +68,44 @@ var mountAllCmd = &cobra.Command{
 	SuggestFor:        []string{"mnta", "mout"},
 	Args:              cobra.ExactArgs(1),
 	FlagErrorHandling: cobra.ExitOnError,
-	Run: func(cmd *cobra.Command, args []string) {
-		_ = VersionCheck()
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if !disableVersionCheck {
+			err := VersionCheck()
+			if err != nil {
+				return err
+			}
+		}
 
 		mountAllOpts.blobfuse2BinPath = os.Args[0]
 		options.MountPath = args[0]
-		processCommand()
+		return processCommand()
 	},
+
 	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return nil, cobra.ShellCompDirectiveDefault
 	},
 }
 
-func processCommand() {
-	parseConfig()
+func processCommand() error {
+	err := parseConfig()
+	if err != nil {
+		return fmt.Errorf("failed to parse config")
```

Comment:
Add err also in the error string. Or we can keep it same as it is in mount.go where we are directly returning err.